dockerfiles/feature-support/hsm-patches/0001-Introduce-generic-keypair-generation-interface-and-e.patch

```diff
@@ -417,36 +443,31 @@ index ec7f279..57cba43 100644
 +	curve_obj = OBJ_nid2obj(curve_nid);
 +	if (!curve_obj)
 +		return -1;
-+	ecdsa_params_len = i2d_ASN1_OBJECT(curve_obj, NULL);
-+	ecdsa_params = (unsigned char *)OPENSSL_malloc(ecdsa_params_len);
-+	if (!ecdsa_params)
++	ec_params_len = i2d_ASN1_OBJECT(curve_obj, NULL);
++	ec_params = (unsigned char *)OPENSSL_malloc(ec_params_len);
++	if (!ec_params)
 +		return -1;
-+	tmp = ecdsa_params;
++	tmp = ec_params;
 +	i2d_ASN1_OBJECT(curve_obj, &tmp);
 +
 +	/* pubkey attributes */
 +	pkcs11_addattr(&pubtmpl, CKA_ID, id, id_len);
 +	if (label)
 +		pkcs11_addattr_s(&pubtmpl, CKA_LABEL, label);
 +	pkcs11_addattr_bool(&pubtmpl, CKA_TOKEN, TRUE);
-+	pkcs11_addattr_bool(&pubtmpl, CKA_DERIVE, FALSE);
-+	pkcs11_addattr_bool(&pubtmpl, CKA_WRAP, FALSE);
++	pkcs11_addattr_bool(&pubtmpl, CKA_DERIVE, TRUE);
```

Comment:
I don't think we need DERIVE on the pubkey. Not sure if this even works.

Reply:
Removed